cli/config.go

```diff
@@ -148,9 +151,17 @@ func getClusters() (map[string]cluster, error) {
 
 	// for all servers created by k3d, get workers and cluster information
 	for _, server := range k3dServers {
-		filters.Add("label", fmt.Sprintf("cluster=%s", server.Labels["cluster"]))
 		clusterName := server.Labels["cluster"]
 
+		if !all && strings.Compare(name, clusterName) != 0 {
```

Comment:
Could as well be `if !all && name != clusterName {...}`

Reply:
Thank you for pointing it out. It is simpler and more readable.  Will fix.

Reply:
Updated.